Include suggested ref and how to select failures (#44)
UBC-MDS/opinionated-practices-for-teaching-reproducibility: manuscript/teaching-reproducibility.Rmd

associated with not adhering to reproducible practices
and having workflows that are opaque and hard to review.

As illustrated in the case we have highlighted above,

> reproducibility errors are not isolated to novices performing irrelevant data analyses.

Reviewer: "irrelevant" is not great here - I think it minimizes the meaning of data analyses performed by novices. I suggest we remove it. 
```suggestion
reproducibility errors are not isolated to novices performing data analyses.
```

> They are occurring at the highest level in both industry and academia

Reviewer: ```suggestion
They also occur in analyses performed by seasoned professionals 
```

and can have substantial real-world consequences.
When selecting failures to share with students,
it can be beneficial to focus on
those that students can easily identify with,
those that have had a big impact,

> and those that come from prolific researchers/companies.

Reviewer: ```suggestion
and those that come from well-known researchers/companies.
```

This is to illustrate that these errors can happen to anyone

> and emphasize that we are not sharing them cast blame,

Author: ```suggestion
and emphasize that we are not sharing them to cast blame,
```

but to highlight the importance of reproducible practices on all levels.

> Another example that illustrates the previous points

Reviewer: I am a bit lost as to what previous points this example is referring to... Not sure if this is because of the new addition, or if we need some more context or reorganization here.

Author: ```suggestion
Another example of an impactful reproducibility mistake in from a well-known research team 
```

is the misreporting of the relationship between public depth and GDP growth,

> which was published in what was regarded as a seminal paper at the time.

Reviewer: Seems like the citation should go here.

Author: ```suggestion
which was published in what was regarded as a seminal paper at the time [@pollin_herndon_2013; @borweinjon_bailey_2013].
```
